beacon-chain/rpc/beaconv1/blocks.go

```diff
@@ -302,13 +344,30 @@ func (bs *Server) ListBlockAttestations(ctx context.Context, req *ethpb.BlockReq
 
 	v1Block, err := migration.V1Alpha1ToV1Block(blk)
 	if err != nil {
-		return nil, status.Errorf(codes.Internal, "Could not convert block to v1")
+		return nil, status.Errorf(codes.Internal, "Could not convert block to v1 block")
 	}
 	return &ethpb.BlockAttestationsResponse{
 		Data: v1Block.Block.Body.Attestations,
 	}, nil
 }
 
+func (bs *Server) signedBeaconBlock(ctx context.Context, block interfaces.SignedBeaconBlock) (*ethpb.SignedBeaconBlock, error) {
```

Comment:
Why is this a pointer receiver? This looks like a conversion or migration method. Is this the most appropriate place to put this method?

Reply:
yeah, it should be in `migration`